articles/vmware/vmw-faq.md

```diff
@@ -376,7 +367,7 @@ UKCloud monitors the underlying platform but doesn't monitor customers' OSs or a
 
 ### Do you offer autoscaling?
 
-There's no standard product offering for autoscale, but the platform API can be used to do this with a little developer effort. UKCloud provides blueprints in the Knowledge Centre on the UKCloud Portal which offer guidance on this.
+There's no standard product offering for autoscale, but the platform API can be used to do this with a little developer effort. UKCloud provides blueprints in the Knowledge Centre which offer guidance on this.
```

Comment:
Blueprints aren't available on the Knowledge Centre at the moment, so can you change this to:
We provide blueprints on the UKCloud web site that offer guidance on this.